server/metrics/metrics.go

```diff
@@ -1532,3 +1571,16 @@ func exponentialBucketRange(min, max, factor float64) []float64 {
 func durationUsecBuckets(min, max time.Duration, factor float64) []float64 {
 	return exponentialBucketRange(float64(min.Microseconds()), float64(max.Microseconds()), factor)
 }
+
+// bucketsWithInterval returns prometheus.Buckets at the given interval, specified in
+// terms of a min and max value, rather than needing to explicitly calculate the
+// number of buckets.
+func bucketsWithInterval(min, max, interval float64) []float64 {
+	buckets := []float64{}
+	current := min
+	for current < max {
+		buckets = append(buckets, current)
+		current += interval
+	}
+	return buckets
+}
```

Comment:
this seems equivalent to `prometheus.LinearBuckets` - https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#LinearBuckets

Reply:
Oh nice - replaced